packages/skeleton-react/src/lib/components/Rating/Rating.tsx

```diff
@@ -1,155 +1,93 @@
-'use client';
+import * as rating from '@zag-js/rating-group';
+import { useMachine, normalizeProps } from '@zag-js/react';
+import { useId, type FC } from 'react';
+import type { RatingProps } from './types';
+import { starEmpty, starHalf, starFull } from '../../internal/nodes';
+import { noop } from '../../internal/noop';
 
-import React, { useCallback, useEffect, useRef, useState } from 'react';
```

Comment:
I've noticed two things while using keyboard navigation.

The first thing is: when no stars are selected, it is impossible to use tab to focus the rating stars (to then add a value with the keyboard).

The second thing is: there is an inconsistency between React and Svelte. Using the keyboard to add values, it will add half a star in React, and a whole star in Svelte.

Reply:
That differs based on the `allowHalf` prop

Reply:
As Hugo said, this is working per my expectation when I test on the React doc examples.

Reply:
Per the two issues:

1. Confirmed - I can replicate this. It may be related to focus states which we plan to return to post-release, but I'll log this specifically so it's not lost in the shuffle.
2. What Hugo said, the stars work per my expectations based on the `allowHalf` setting on the doc examples.

Reply:
Per 1, I've added this to the list in the original post above. I'll be making dedicated tickets for these on GitHub soon.